### ISSUE TITLE
Fix z19 water boundary test

### ISSUE THREAD

**Issue by zerebubuth** [tilezen/vector-datasource#757]
This [test](https://github.com/mapzen/vector-datasource/blob/cf223a5c72093876e0e5012df438e424fd53d026/test/502-water-boundaries-slow.py#L7) fails on dev, where tiles are cut at z16, but not locally where they're cut at z19. This means there's a bug, perhaps with the precision of the cutting / boundary detection, but it's unclear exactly where that is.

**Comment by nvkelso** [comment]
@zerebubuth Does this need to be fixed in the v0.10 release?

**Comment by zerebubuth** [comment]
Everything else for v0.10 is done, I'd rather not delay the release by tracking this down.

It's an LOE high, because we don't know where this problem is coming from, and haven't yet put the potentially significant amounts of work into reproducing it locally. It's the bug we discussed a while back where the z16 tile seems fine, but when it's cut to z19 then this error appears, which is pretty perplexing since the z19 tile ought to be cut from the z16 one!

IIRC, we punted on it for previous releases, so I think it's safe to do the same again. And there are other water boundary tests which continue to pass.
